assets/css/style.css

```diff
@@ -0,0 +1,223 @@
+    color: whitesmoke;
+    padding-top: 15px;
+    margin-bottom: 2px solid rgb(36, 92, 156);
+    text-decoration: none;
+    text-align: center;
+    font-weight: bold;
+    border-bottom: 4px solid grey;
+}
+
+#home {
+    text-decoration: none;
+    background-color: rgb(63, 62, 62);
+    color: whitesmoke;
+    padding-top: 20px;
+}
+
+.aboutme {
+    grid-column: 2/5;
+    grid-row: 2/3;
+    text-align: left;
+    font-size: 20px;
+}
+
+.content {
+    grid-column: 2/5;
+    grid-row: 4/5;
+    text-align: left;
+    background-color: rgb(238, 238, 238);
+    box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
+}
+
+.hangman {
+    text-align: left;
+    background-color: rgb(238, 238, 238);
+    font-size: 60px;
+    align-content: center;
+    text-align: center;
+    padding: 15px;
+    box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
+}
+
+.rpggame {
+    text-align: left;
+    background-color: rgb(238, 238, 238);
+    font-size: 60px;
+    align-content: center;
+    text-align: center;
+    padding: 15px;
+    box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
+}
+
+.triviagame {
+    text-align: left;
+    background-color: rgb(238, 238, 238);
+    font-size: 60px;
+    align-content: center;
+    text-align: center;
+    padding: 15px;
+    box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
+}
+
+.rutgersinfowidget {
+    text-align: left;
+    background-color: rgb(238, 238, 238);
+    font-size: 60px;
+    align-content: center;
+    text-align: center;
+    padding: 15px;
+    box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
+}
+
+.rockpaperscissors {
+    text-align: left;
+    background-color: rgb(238, 238, 238);
+    font-size: 60px;
+    align-content: center;
+    text-align: center;
+    padding: 15px;
+    box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
+}
+
+.contact {
+    grid-column: 2/5;
+    grid-row: 2/3;
+    text-align: left;
+    font-size: 20px;
+}
+
+.portfolio {
+    grid-column: 2/5;
+    grid-row: 2/3;
+    text-align: left;
+    font-size: 20px;
+}
+
+form {
+    grid-column: 2/5;
+    grid-row: 4/5;
+    background-color: rgb(238, 238, 238);
+    padding: 20px;
+    box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
+}
+
+.frontnav {
+    grid-column: 1/2;
+    grid-row: 1/2;
+    background-color: rgb(26, 25, 25);
+}
+
+.middlenav {
+    grid-column: 3/4;
+    grid-row: 1/2;
+    background-color: rgb(26, 25, 25);
+}
+
+a:hover { 
+    color:grey;
+}
+
+h1 {
+    margin: 0;
+    padding: 5px;
+}
+
+img {
+    margin-right: 5px;
+}
+
+p {
+    font-size: 22px;
+    margin: 20px;
+}
+
+.socialmedia {
+    grid-column: 2/5;
+    grid-row: 5/6;
+    display: flex;
+    align-items: flex-end;
+    justify-content: flex-end;
+}
+
+.findme {
+    margin-right: 10px;
+    font-size: 20px;
+}
+
+#footer {
```

Comment:
don't use ids for styling
https://stackoverflow.com/questions/8084555/why-selecting-by-id-is-not-recommended-in-css